CI/vale/vale_styles/proselint/Diacritical.yml

```diff
@@ -0,0 +1,96 @@
+extends: substitution
+message: Consider using '%s' instead of '%s'.
+ignorecase: true
+level: error
+action:
+  name: replace
+swap:
+  beau ideal: beau idéal
+  boutonniere: boutonnière
+  bric-a-brac: bric-à-brac
+  cafe: café
+  cause celebre: cause célèbre
+  chevre: chèvre
+  cliche: cliché
+  consomme: consommé
+  coup de grace: coup de grâce
+  crudites: crudités
+  creme brulee: crème brûlée
+  creme de menthe: crème de menthe
+  creme fraice: crème fraîche
+  creme fresh: crème fraîche
+  crepe: crêpe
+  debutante: débutante
+  decor: décor
+  deja vu: déjà vu
+  denouement: dénouement
+  facade: façade
+  fiance: fiancé
+  fiancee: fiancée
+  flambe: flambé
+  garcon: garçon
+  lycee: lycée
+  maitre d: maître d
+  menage a trois: ménage à trois
+  negligee: négligée
+  protege: protégé
+  protegee: protégée
+  puree: purée
+  my resume: my résumé
+  your resume: your résumé
+  his resume: his résumé
+  her resume: her résumé
+  a resume: a résumé
+  the resume: the résumé
+  risque: risqué
+  roue: roué
+  soiree: soirée
+  souffle: soufflé
+  soupcon: soupçon
+  touche: touché
+  tete-a-tete: tête-à-tête
+  voila: voilà
+  a la carte: à la carte
+  a la mode: à la mode
+  emigre: émigré
+
+  # Spanish loanwords
+  El Nino: El Niño
+  jalapeno: jalapeño
+  La Nina: La Niña
+  pina colada: piña colada
+  senor: señor
+  senora: señora
+  senorita: señorita
+
+  # Portuguese loanwords
+  acai: açaí
+
+  # German loanwords
+  doppelganger: doppelgänger
+  Fuhrer: Führer
+  Gewurztraminer: Gewürztraminer
+  vis-a-vis: vis-à-vis
+  Ubermensch: Übermensch
+
+  # Swedish loanwords
+  filmjolk: filmjölk
+  smorgasbord: smörgåsbord
+
+  # Names, places, and companies
+  Beyonce: Beyoncé
+  Bronte: Brontë
+  Bronte: Brontë
```

Comment:
duplication?

Reply:
It's an imported file, not written by us, I did not check that... Si I will ignore ;)